.github/workflows/publish-cdn.yml

```diff
@@ -4,29 +4,9 @@ on:
   workflow_dispatch:
 
 jobs:
-  test:
-    runs-on: ubuntu-latest
-    container: node:14
-    env:
-      NEWRELIC_ENVIRONMENT: ci
-      JIL_SAUCE_LABS_USERNAME: ${{ secrets.JIL_SAUCE_LABS_USERNAME }}
-      JIL_SAUCE_LABS_ACCESS_KEY: ${{ secrets.JIL_SAUCE_LABS_ACCESS_KEY }}
-      NEW_RELIC_LICENSE_KEY: ${{ secrets.JIL_NODE_NEW_RELIC_LICENSE_KEY }}
-      BUILD_NUMBER: Release${{ github.event.number }}-${{ github.run_number }}
-      AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
-      AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-    steps:
-      - uses: actions/checkout@v3
-      - name: install
-        run: npm ci
-      - name: build:all
-        run: npm run build:all
-      - name: run tests
-        run: node ./tools/jil/bin/cli.js -f merged -b chrome@latest -s -t 85000 --concurrent=10
   publish-to-s3:
     runs-on: ubuntu-latest
     container: node:14
-    needs: [test]
```

Comment:
Just verifying here we are intentionally removing the smoke tests for the cdn publish.

Reply:
This is meant to be a "standalone" GHA only to be used to "re-run" if something goes wrong in the CI tied to cutting a release.